e2e/__tests__/jestCircus.test.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+import runJest from '../runJest';
+import {countExecutedTests} from '../Utils';
+
+test('run only "it.only" tests', () => {
+  const jestCircusPath = require.resolve('../../packages/jest-circus/runner');
```

Comment:
All E2E tests run with both circus and jasmine (on ci), so you don't need to supply a runner. That way we verify it behaves the same with both runners :)

Reply:
So maybe name the integration test `focused-tests` or something?